graypy/handler.py

```diff
@@ -183,6 +183,8 @@ def make_message_dict(record, debugging_fields, extra_fields, fqdn, localname,
     logging.INFO: 6,
     logging.DEBUG: 7,
 }
+if hasattr(logging, 'NOTICE'):
```

Comment:
TBH, if the user is ok with `verboselogs` monkey-patching the standard logging lib they should be fine with updating the `SYSLOG_LEVELS` themselves:
```
import graypy.handler
graypy.handler.SYSLOG_LEVELS[logging.NOTICE] = 5
```
Unless you feel differently, I suggest we close this PR.